OmniKit/PumpManager/PodCommsSession.swift

```diff
@@ -175,6 +175,21 @@ public class PodCommsSession {
         }
     }
 
+    // pad the given message to be more than one packet using GetStatus sub-messages
+    private func padMessage(message: [MessageBlock]) -> [MessageBlock] {
+        let packetLen = PacketType.pdm.maxBodyLen()
+        let getStatusCommand = GetStatusCommand()
+        let getStatusLen = 3
+
+        var paddedMessage = message
+        var encodedLen = messageLength(message: message)
+        while encodedLen <= packetLen {
```

Comment:
Are there commands short enough to need more than one status added to them to reach two packets?

Reply:
The following nonce commands all need multiple 3-byte get status sub-commands to pad to reach two packets: 
$08 command is only 7 bytes long, the $11 command is 6 bytes long & the $1F command is 6 bytes long (the $1C deactivate pod command is only 5 bytes, but it can't be padded). The total encoded MessageBlock length has to exceed 25 bytes (= 31 - 4 [address] - 1 [B9] - 1 [Blen]) to not fit in a single packet.